docs/software/databases.md

```diff
@@ -34,10 +34,58 @@ Source: [MariaDB](https://mariadb.com/), [LGPL](https://commons.wikimedia.org/w/
     - Username = `root`
     - Password = The same as your root login password, default is `dietpi`
 
+=== "Configuration"
+
+    <h2>Directories</h2>
+
+    - Database files: `/mnt/dietpi_userdata/mysql`
+    - Configuration files: `/etc/mysql`
+
+    <h2>Configuration files</h2>
+
+    The MariaDB/MySQL tools read configuration files in the following order:
+    
+    1. `/etc/mysql/my.cnf` symbolic links to this file, reason why all the rest is read.
+    1. `/etc/mysql/mariadb.cnf` (this file) to set global defaults.
+    1. `/etc/mysql/conf.d/*.cnf` to set global options.
+    1. `/etc/mysql/mariadb.conf.d/*.cnf` to set MariaDB-only options.
+    1. `~/.my.cnf` to set user-specific options.
+    
+    As a best practice, it is recommended to place MariaDB related user configuration files into `/etc/mysql/mariadb.conf.d/*.cnf` and to name the files in the manner of `<2-digit-number>-<configurationname>.cnf` (example: `99-myMariaDB_config.cnf`).
+
+    <h2>Configuration example (for a Nextcloud instance)</h2>
+
+    ```
+    [mysqld]
+    # Disable file logging in favour of "journalctl -u mariadb"
+    skip_log_error=0
+    # Disable host name resolving
+    # NB: For remote access, IP addresses then need to be used when creating/altering database users.
+    skip_name_resolve=1
+    # Disable the TCP listener all together
+    # NB: For any database access, the UNIX socket "/run/mysqld/mysqld.sock" then needs to be used.
+    skip_networking=1
+    # The query cache is not recommended on multi-core machines.
+    query_cache_size=0
+    query_cache_type=0
+    # Since we use no MyISAM tables, disable key buffer
+    key_buffer_size=0
+    # Since we have only 336 KiB Aria tables, limit Aria page cache to 512 KiB
+    aria_pagecache_buffer_size=512k
+    # Since max 8 concurrent connections were reported by MySQLTuner, max connections are limited to 16.
+    max_connections=16
+    # All InnoDB databases are around 30 MiB, hence a buffer of 64 MiB is sufficient, including future growth.
+    innodb_buffer_pool_size=64M
+    # MySQLTuner recommends 25% InnoDB buffer size as log file size.
+    innodb_log_file_size=16M
+    ```
```

Comment:
I am a bit afraid here that people might just copy&paste this example in cases it can break or worsen things. Even for a Nextcloud-only instance, the last 3 settings depend on NC usage, and especially the last 2 depend also on 32-bit vs 64-bit. E.g. on my home server, where above example is coming from, I quadrupled the numbers in the meantime, one doubling needed only for 32-bit => 64-bit switch:
```ini
innodb_buffer_pool_size=256M
innodb_log_file_size=64M
```
Maybe better to explain some values which are typically safe to apply for instances which  are not used by remote machines, and some others only in the context of monitoring actual database usage with e.g. MySQLTuner or direct database sys/performance schema queries.